src/isValidNumber.js

```diff
@@ -0,0 +1,26 @@
+import { either } from 'ramda';
+
+import isFloat from './isFloat';
+import isInteger from './isInteger';
+
+/* eslint-disable max-len */
```

Comment:
Drop this. Your lines length should not go over 100 chars

Reply:
Not sure what you mean.  What should I drop?

Reply:
```javascript
/* eslint-disable max-len */